tests/wpt/metadata/resource-timing/idlharness.any.js.ini

```diff
@@ -97,9 +91,6 @@
   [PerformanceResourceTiming interface: resource must inherit property "domainLookupStart" with the proper type]
     expected: FAIL
 
-  [PerformanceResourceTiming interface: resource must inherit property "connectStart" with the proper type]
```

Comment:
Let's revert this change and the next one in this file. That should allow us to merge this PR.

Reply:
I've pushed the revert to fix this.